reflection/serverreflection_test.go

```diff
@@ -78,65 +84,43 @@ func loadFileDesc(filename string) (*dpb.FileDescriptorProto, []byte) {
 	return fd, b
 }
 
+func loadFileDescDynamic(filename string) (*dpb.FileDescriptorProto, []byte) {
+	p := &protoparse.Parser{}
+	fds, err := p.ParseFiles(filename)
```

Comment:
This will need to call `testdata.Path(filename)` to avoid potential problems caused by file path.